config/routes.rb

```diff
@@ -23,6 +23,6 @@
     get '/dashboard', to: 'users#show'
     resources :stations, only: [:update, :edit, :destroy, :new, :create]
     resources :conditions, only: [:new, :create, :edit, :update, :destroy]
-    resources :trips, only: [:new, :create]
+    resources :trips, only: [:new, :create, :edit, :show, :update, :destroy]
```

Comment:
Quick question on this section, it looks like we have a :show for admin_trips in the config but I'm not seeing anything in the admin trip controller for show. Do we need to have this here or will we need it later?

Reply:
All I know is that when you delete it, the tests fail.

Reply:
Seems like a good enough answer to me. Maybe something worth circling back to later to see why it's failing without the show.